docs/time_series_causal_graph.md

```diff
@@ -0,0 +1,420 @@
+# Time Series Causal Graph
+The class `cai_causal_graph.causal_graph.TimeSeriesCausalGraph`, inheriting directly from `cai_causal_graph.causal_graph.CausalGraph`,
+extends causal graphs to time series data.
+
+For the theoretical background of causality for time series, please refer to Chapter 10 of _Peters, J., Janzing, D. and Schölkopf, B., 2017. Elements of causal inference: foundations and learning algorithms (p. 288). The MIT Press_. 
+
+The main addition of `cai_causal_graph.causal_graph.TimeSeriesCausalGraph`
```

Comment:
Incorrect link throughout. It is `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph`. Same for `Node` and `TimeSeriesNode`.